public/controllers/management/components/management/mg-logs/logs.js

```diff
@@ -85,7 +85,7 @@ export default compose(
 
     async componentDidMount() {
       try {
-        this.height = window.innerHeight - this.offset;
+        this.F = window.innerHeight - this.offset;
```

Comment:
is this for something?

Reply:
Check the actual file, for some reason Github doesn't show the latest version.

Reply:
when I clone the branch `this.F` appeared

Reply:
Solved. Please review again.